modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/IgniteTxCacheWriteSynchronizationModesMultithreadedTest.java

```diff
@@ -205,121 +341,111 @@ private void multithreaded(CacheWriteSynchronizationMode syncMode,
+                commitMultithreaded((ignite1, cache) -> {
                     ThreadLocalRandom rnd = ThreadLocalRandom.current();
 
-                    Map<Integer, Integer> map = new TreeMap<>();
+                    Map<Integer, Integer> map = new LinkedHashMap<>();
 
-                    for (int i = 0; i < 100; i++) {
+                    for (int i = 0; i < 10; i++) {
                         Integer key = rnd.nextInt(MULTITHREADED_TEST_KEYS);
 
                         map.put(key, rnd.nextInt());
                     }
 
-                    try {
-                        try (Transaction tx = ignite.transactions().txStart(PESSIMISTIC, REPEATABLE_READ)) {
+                    while (true) {
+                        try (Transaction tx = ignite1.transactions().txStart(OPTIMISTIC, SERIALIZABLE)) {
                             for (Map.Entry<Integer, Integer> e : map.entrySet())
                                 cache.put(e.getKey(), e.getValue());
 
                             tx.commit();
-                        }
-                    }
-                    catch (CacheException | IgniteException ignored) {
-                        // No-op.
-                    }
-                }
-            });
 
-            if (!MvccFeatureChecker.forcedMvcc()) {
-                commitMultithreaded(new IgniteBiInClosure<Ignite, IgniteCache<Integer, Integer>>() {
-                    @Override public void apply(Ignite ignite, IgniteCache<Integer, Integer> cache) {
-                        ThreadLocalRandom rnd = ThreadLocalRandom.current();
-
-                        Map<Integer, Integer> map = new LinkedHashMap<>();
-
-                        for (int i = 0; i < 10; i++) {
-                            Integer key = rnd.nextInt(MULTITHREADED_TEST_KEYS);
-
-                            map.put(key, rnd.nextInt());
+                            break;
                         }
-
-                        while (true) {
-                            try (Transaction tx = ignite.transactions().txStart(OPTIMISTIC, SERIALIZABLE)) {
-                                for (Map.Entry<Integer, Integer> e : map.entrySet())
-                                    cache.put(e.getKey(), e.getValue());
-
-                                tx.commit();
-
-                                break;
-                            }
-                            catch (TransactionOptimisticException ignored) {
-                                // Retry.
-                            }
-                            catch (CacheException | IgniteException ignored) {
-                                break;
-                            }
+                        catch (TransactionOptimisticException ignored) {
+                            // Retry.
+                        }
+                        catch (CacheException | IgniteException ignored) {
+                            break;
                         }
                     }
                 });
```

Comment:
Why we still keep section with cache destroy?
Hang on destroy was the reason of suite timeout previous time.
Let's get rid of destroy, nodes are stopped after test anyway.